app/lib/logic/rolling_updates.rb

```diff
@@ -284,6 +284,15 @@ def missing_config
       end
     end
 
+    module User
+      def provider_can_use?(feature)
+        return false if Logic::RollingUpdates.skipped?
```

Comment:
https://github.com/3scale/porta/pull/593/files#r258178494

Reply:
Yes not needed